airbyte-webapp/src/packages/cloud/experiments/RevealSources.ts

```diff
@@ -0,0 +1,59 @@
+import { ServiceDropdownOption } from "views/Connector/ServiceForm/components/Controls/ConnectorServiceTypeControl/utils";
+
+export const sortRevealSources = (tech: string[] = [], sources: ServiceDropdownOption[]) => {
+  const sourcesIndex: Record<string, string> = {
```

Comment:
Suggestion: 
I would define this object outside of the function. I think it's a variable that doesn't need to be defined at each function execution.

```suggestion
  const SOURCE_ID_CLEARBIT_MAP: Record<string, string> = {...}
  export const sortRevealSources = (tech: string[] = [], sources: ServiceDropdownOption[]) => {
  ...
  }
```